controllers/suggestions/suggest_match_video_controller.py

```diff
@@ -42,11 +43,26 @@ def post(self):
         self._require_login()
 
         match_key = self.request.get("match_key")
-        youtube_url = self.request.get("youtube_url")
-        youtube_id = YouTubeVideoHelper.parse_id_from_url(youtube_url)
-
-        status = SuggestionCreator.createMatchVideoYouTubeSuggestion(self.user_bundle.account.key, youtube_id, match_key)
-
+        video_url = self.request.get("video_url")
+        
+        if any(s in video_url for s in MediaParser.YOUTUBE_URL_PATTERNS):
+            youtube_id = YouTubeVideoHelper.parse_id_from_url(video_url)
+
+            status = SuggestionCreator.createMatchVideoYouTubeSuggestion(self.user_bundle.account.key, youtube_id, match_key)
+        elif any(s in video_url for s in MediaParser.VIMEO_URL_PATTERNS):
+            vimeo_id = None
```

Comment:
I tried to simplify the code I added for the social links, you can see how I did it here: https://github.com/the-blue-alliance/the-blue-alliance/pull/1389/files#diff-40f412c989d5abffbb74ac686847c330R61